.github/workflows/gen-docs.yaml

```diff
@@ -0,0 +1,29 @@
+name: Generate all files related to documentation
+
+on:
+  workflow_call:
+
+jobs:
+  genereate-markdown-pages:
+    name: Generate CLI Usage page
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install python and set up Poetry
+        uses: bakdata/ci-templates/actions/python-setup-poetry@v1.5.3
+        with:
+          poetry-version: "1.3.2"
+          python-version: "3.10"
+
+      - name: Install optional dependencies
+        shell: bash
+        run: poetry install --with docs
+
+      - name: Generate CLI Usage page
+        shell: bash
+        run: |
+          # Generate page with typer-cli
+          poetry run typer ./kpops/cli/main.py utils docs --name kpops --output ./docs/docs/user/references/cli-commands.md
+          # Fix title
+          sed -i '1s/.*/# CLI Usage/' ./docs/docs/user/references/cli-commands.md
```

Comment:
Who commits the changes of the file?

Reply:
Very good point :DD

I've fixed this I believe.